db/versions/0070.yml

```diff
@@ -0,0 +1,248 @@
+version: 70
```

Comment:
May be helpful to compare to https://github.com/taskcluster/taskcluster/blob/main/db/versions/0013.yml.